prime-dotnet-webapi/Services/SiteRegistrationService.cs

```diff
@@ -0,0 +1,35 @@
+using Prime.Models.Api;
+using Prime.Engines;
+using Prime.Models;
+
+namespace Prime.Services
+{
+    public class SiteRegistrationService : ISiteRegistrationService
```

Comment:
The Services in our app are registered with the Dependency Injection framework to handle creating dependencies. This does not have any dependencies, so isn't really a service and has no benefit being registered as such.

Reply:
> The Services in our app are registered with the Dependency Injection framework to handle creating dependencies. This does not have any dependencies, so isn't really a service and has no benefit being registered as such.

Similar reason for the service/engine pattern. can remove the service layer and call the engine directly.